std/php/PDO.hx

```diff
@@ -19,85 +19,36 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
  */
-package php.db;
+package php;
```

Comment:
Is it really needed to move this module to another package?
Looks like not much of a benefit comparing to introducing a breaking change.